### PR TITLE
Converting ci-pa11y to drydock pattern.

### PR DESCRIPTION
**Pull request by acabouet**

### CODE REVIEW

**Comment by acabouet** [comment]
@grugnog where I'm at.

**Comment by acabouet** [comment]
Plus this: https://git.civicactions.net/msf/msf/merge_requests/722

File: Dockerfile [hunk 1]
```diff
@@ -29,8 +33,10 @@ RUN yarn install --silent --production --non-interactive && \
     yarn cache clean --force
 COPY entrypoint.sh ./
 ENV PATH="/home/node/node_modules/.bin:${PATH}"
-WORKDIR /home/node/app
+
+COPY .pa11yci /home/node/app/
```
Comment (acabouet) [inline Dockerfile]: Can't figure out how to make this do the thing/why it's not working.

Comment (grugnog) [reply Dockerfile]: This looks correct - is the .pa11yci file in the same dir as the Dockerfile?

Comment (acabouet) [reply Dockerfile]: Yes it is

**Pull request merged**